distribution/sdk/sample-soap-endpoint/src/main/resources/OSGI-INF/permissions.perm

```diff
@@ -0,0 +1,17 @@
+(java.io.FilePermission "-" "read,write,delete")
```

Comment:
✏️ Nit: It would be good to group these permissions together. In fact in general, if permissions could be sorted where possible, it would probably help when reviewing and editing them.